microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1010,21 +1008,21 @@ See also:
 
 
 <!-- ------------------------------ -->
-#### Controller's script locale
+#### Locale for Javascript
```

Comment:
change to:
Locale for JavaScript

Reply:
via v111 PR, deleted section and added its link in Browser > Language section instead.